### ISSUE TITLE
[A11Y] Include the button in a landmark

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/back_to_top#21]
An accessibility review flagged that this button should be included in a landmark. I see a parallel issue over in the D10 queue:
 - https://www.drupal.org/project/back_to_top/issues/3254713
> An accessibility best practice is to include all perceivable content in a landmark: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/index.html . This is not the case for the "back to top" button: since the button is just appended to the body, there is no landmark set on it. And since it's added via Javascript, it's not possible to add the wrapper via template either.
> ## Steps to reproduce
> Add and enable the "back to top" module to a website, then run an accessibility checker such as Siteimprove's accessibility checker. The issue will be flagged.
> ## Proposed resolution
> Wrap the button in a div with an appropriate role, and set an aria label to it.

**Comment by laryn** [comment]
Fixed in https://github.com/backdrop-contrib/back_to_top/commit/fd2e03c2227fd2c022a2d92c3a4c509719567cec
